arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -23,178 +23,227 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "The specified failover group does not exist."
           }
         },
-        "deprecated": false,
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete failover group": {
+            "$ref": "../examples/FailoverGroupDelete.json"
+          }
+        }
       },
       "patch": {
         "tags": [
           "FailoverGroups"
         ],
         "description": "Updates a failover group.",
         "operationId": "FailoverGroups_Update",
-        "x-ms-examples": {
-          "Update failover group": {
-            "$ref": "../examples/FailoverGroupUpdate.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
nit: maintain a consistent parameter ordering, either place ApiVersion and SubscriptionId together at the beginning of the parameters section or at the end.